hack/generated/apis/microsoft.resources/v1alpha1api20200601/resourcegroup_types.go

```diff
@@ -127,6 +137,54 @@ func (status *ResourceGroupStatus) PopulateFromARM(owner genruntime.KnownResourc
 	return nil
 }
 
+func (status *ResourceGroupStatus) ConvertStatusTo(destination genruntime.ConvertibleStatus) error {
```

Comment:
minor: Missing comments here and below too?

Reply:
Added.